README.md

```diff
@@ -67,11 +67,11 @@ The master branch works with **PyTorch 1.5** or higher.
 
```

Comment:
Maybe we can add AD for 1.0.0 version, like MMClassification https://github.com/open-mmlab/mmclassification